### PR TITLE
exploitmon: add new plugin

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

**Comment by skvl** [comment]
I will fix this as soon as possible.

**Comment by skvl** [comment]
Fixed.

One could catch event like this (with `-o kv` option):
```
exploitmon Time=1617702275.324506,PID=2836,PPID=2788,TID=2840,ProcessName="\Device\HarddiskVolume2\Users\John\Desktop\share\example.exe",Method=NtTerminateProcess,Method="Token modification detected"
```

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by skvl** [comment]
@tklengyel what's the error with debian test?

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

File: src/plugins/exploitmon/exploitmon.h [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
```
Comment (tklengyel) [inline src/plugins/exploitmon/exploitmon.h:1]: Please re-paste the license from the latest LICENSE file, there was a typo that got fixed that's not included in here

Comment (skvl) [reply src/plugins/exploitmon/exploitmon.h:1]: Done

**Comment by skvl** [comment]
@tklengyel, I have re-based the branch

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**